### PR TITLE
Fixes for issues #4 and #6

### PR DESCRIPTION
**Pull request by k33rs**
Solved issues [#4][1] and [#6][2].

[1]:[https://github.com/martel-innovate/eWine-connectivity-manager/issues/4]
[2]:[https://github.com/martel-innovate/eWine-connectivity-manager/issues/6]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,12 @@
 
 The app exposes a REST API to control wifi connectivity. Using the right network interface, it is possible to connect and disconnect from wifi networks, but also store and manage network configurations in /etc/network/interfaces and in a Sqlite3 database.
 
+The app runs with [Python 2.7.9][1], but ideally should work with [all Python 2.7 releases][2].
```
Comment (taliaga) [inline README.md]: It's enough to say 2.7.
We'll double-check then that we also support python3, but don't worry now about it.

File: wifi_manager/core.py [hunk 2]
```diff
@@ -194,13 +193,14 @@ def do_nothing():
     raise WifiException(e.message, 500)
 
 
-def delete(iface, ssid, db):
+def delete(iface, ssid, db, db_only=False):
```
Comment (taliaga) [inline wifi_manager/core.py:196]: Let's talk and think about why this flag is needed :)

**Review by taliaga — COMMENTED** [review]
All in all... good!, this is so much better than the "collector branch" :)

File: wifi_manager/core.py (hunk 2 above)
Comment (k33rs) [reply wifi_manager/core.py:196]: So shall I merge and look later into mocking?

**Comment by taliaga** [comment]
As discussed,feel free to merge

**Pull request merged**